tests/tool_excimer_profile_test.php

```diff
@@ -33,6 +33,8 @@ class tool_excimer_profile_test extends \advanced_testcase {
     protected function setUp(): void {
         parent::setUp();
         $this->resetAfterTest();
+        profile_helper::init();
```

Comment:
Why are these required. It doesnt look like you are doing anything new in your tests

Reply:
Its for #312 - Because otherwise the tests fail because the static vars are not populated yet https://github.com/catalyst/moodle-tool_excimer/blob/5257c205ba836337b01b9f5f2780efea73332bcc/classes/profile_helper.php#L34-L36

I split it into a separate commit 83c3311d0509c562ec3793f73d3b2aef64f5098d but kept it in this Pr since its a tiny change